docs/root/version_history/current.rst

```diff
@@ -44,6 +44,7 @@ New Features
 * access_log: added new access_log command operators to retrieve upstream connection information: ``%UPSTREAM_PROTOCOL%``, ``%UPSTREAM_PEER_SUBJECT%``, ``%UPSTREAM_PEER_ISSUER%``, ``%UPSTREAM_TLS_SESSION_ID%``, ``%UPSTREAM_TLS_CIPHER%``, ``%UPSTREAM_TLS_VERSION%``, ``%UPSTREAM_PEER_CERT_V_START%``, ``%UPSTREAM_PEER_CERT_V_END%`` and ``%UPSTREAM_PEER_CERT%``.
 * dns_resolver: added :ref:`include_unroutable_families<envoy_v3_api_field_extensions.network.dns_resolver.apple.v3.AppleDnsResolverConfig.include_unroutable_families>` to the Apple DNS resolver.
 * ext_proc: added support for per-route :ref:`grpc_service <envoy_v3_api_field_extensions.filters.http.ext_proc.v3.ExtProcOverrides.grpc_service>`.
+* http2: added initial implementation for deferring processing of backed up http2 streams by buffering in the receiving codec. This is guarded by ``envoy.reloadable_features.defer_processing_backedup_streams``, but is disabled by default. Trailers and bodies are buffered if a stream is backed up. Body data is processed in chunks up to the size of the receiving codec's buffer high watermark. Fairness is implemented by prioritizing streams that haven't written to the connection recently to process deferred data when we transition from network connection backed up to network connection low watermark.
```

Comment:
I suspect his applies to both upstream and downstream connections. Is that right?